cmd/jujud/agent/machine/manifolds.go

```diff
@@ -797,7 +799,29 @@ func Manifolds(config ManifoldsConfig) dependency.Manifolds {
 			NewFacade:     credentialvalidator.NewFacade,
 			NewWorker:     credentialvalidator.NewWorker,
 		}),
+
+		// TODO: (hml) 2018-07-17
+		// Remove when upgrade-series feature flag removed.
+		//upgradeSeriesEnabledName: featureflag.Manifold(featureflag.ManifoldConfig{
+		//	StateName: stateName,
+		//	FlagName:  feature.UpgradeSeries,
+		//	Invert:    false,
+		//	Logger:    loggo.GetLogger("juju.worker.upgradeseries.enabled"),
+		//	NewWorker: featureflag.NewWorker,
+		//}),
```

Comment:
Should we keep code comments?

Reply:
I think we should keep these ones; it's not dead code so much as nascent.